<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-HQBDKGDN" failures="1" tests="4" name="com.crm.qa.tests.HomePageTest" time="142.276" errors="1" timestamp="2021-11-26T13:16:36 IST" skipped="2">
  <testcase classname="com.crm.qa.tests.HomePageTest" name="verifyHomePageTitle" time="36.828">
    <failure message="Title doen&#039;t match expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Title doen't match expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.crm.qa.tests.HomePageTest.verifyHomePageTitle(HomePageTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- verifyHomePageTitle -->
  <system-out/>
  <testcase classname="com.crm.qa.tests.HomePageTest" name="verifyHomePageTitle" time="38.663">
    <skipped/>
  </testcase> <!-- verifyHomePageTitle -->
  <system-out/>
  <testcase classname="com.crm.qa.tests.HomePageTest" name="clickOnContactsLink" time="32.513">
    <error message="No frame element found by name or id mainpanel
Build info: version: &#039;4.0.0&#039;, revision: &#039;3a21814679&#039;
System info: host: &#039;LAPTOP-HQBDKGDN&#039;, ip: &#039;192.168.150.102&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;16.0.2&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchFrameException">
      <![CDATA[org.openqa.selenium.NoSuchFrameException: No frame element found by name or id mainpanel
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: 'LAPTOP-HQBDKGDN', ip: '192.168.150.102', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.2'
Driver info: driver.version: unknown
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.frame(RemoteWebDriver.java:989)
at com.crm.qa.util.TestUtil.switchToFrame(TestUtil.java:23)
at com.crm.qa.tests.HomePageTest.clickOnContactsLink(HomePageTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </error>
  </testcase> <!-- clickOnContactsLink -->
  <system-out/>
  <testcase classname="com.crm.qa.tests.HomePageTest" name="clickOnContactsLink" time="34.272">
    <skipped/>
  </testcase> <!-- clickOnContactsLink -->
  <system-out/>
</testsuite> <!-- com.crm.qa.tests.HomePageTest -->
